technical-reports/format/groups.md

```diff
@@ -80,7 +80,7 @@ Group keys without a dollar sign (`$`) prefix denote:
       "$description": "This is an example of a group containing a single token",
       "Token name": {
         "$value": "#000000",
```

Comment:
Here, and below, all colors should match the updated format:

```suggestion
        "$value": {
          "colorSpace": "srgb",
          "channels": [0, 0, 0]
        }
```

I think we don’t need `hex` on these, too, because we’re just specifying sRGB colors (however, if using a color gamut or bit depth higher than 8-bit sRGB, it could be provided).